src/main/resources/application.properties

```diff
@@ -28,10 +30,7 @@ waitlist.collection.name: wait_list
 waitlist.collection.ttlValue: 600
 waitlist.initialDelayResend: 2000
 waitlist.fixedRateResend: 15000
-database.name: eiffel_intelligence_eiffelxxx
-subscription.collection.name: subscription
-
-server.port: 8090
```

Comment:
As I mentioned above these properties should be here as they are. Since the component will not find right properties to inject and camel case is not for use in properties naming.

Reply:
2 different DBs and collection are used in the EI backend for subscription CRUD & Subscription handling. So updated are made in DB eiffel_intelligence_eiffelxxx collection: subscription and subscription handling is performed in DB: subscription and collection: subscription. Will remove DB subscription with collection:subscription. And update subscription handler to DB:  database.name & subscription.collection.name. Will remove subscriptionDataBaseName & subscriptionCollectionName